backend/app/usecase/url/urlcreator_test.go

```diff
@@ -117,6 +113,26 @@ func TestURLCreatorPersist_CreateURL(t *testing.T) {
 				CreatedAt:   &utc,
 			},
 		},
+		{
+			name: "automatically generate alias if empty string alias provided",
+			urls: urlMap{},
+			availableKeys: []external.Key{
+				"test",
+			},
+			alias: &emptyAlias,
```

Comment:
Let's hard code empty alias here.

Reply:
I did it this way because you can't pass the address of a string literal into a string pointer variable. See the following:
https://stackoverflow.com/questions/61004337/cannot-convert-untyped-string-constant-to-string
https://stackoverflow.com/questions/42594789/initialize-string-pointer-in-struct
We would have to update `CreateURL` method to accept `string` literal instead (which I believe is represented as an address anyways in Go) and `URLInput`'s `CustomAlias` field should also be updated to be `string` literal.